src/XrdApps/XrdMpxStats.cc

```diff
@@ -473,6 +473,7 @@ int main(int argc, char *argv[])
                  else {Say.Emsg(":", "Invalid format - ", optarg); Usage(1);}
                  break;
        case 'h': Usage(0);
+                 break;
```

Comment:
`Usage(0)` will call `exit(0)`, but adding `break` is an extra failsafe that shouldn't hurt.